test/dynamo/test_misc.py

```diff
@@ -92,11 +92,27 @@
 from torch.testing._internal.logging_utils import logs_to_string
 
 
+pytree_modules = {
+    "python": python_pytree,
+}
 if python_pytree._cxx_pytree_dynamo_traceable:
     import torch.utils._cxx_pytree as cxx_pytree
+
+    pytree_modules["cxx"] = cxx_pytree
 else:
     cxx_pytree = None
 
+
+try:
+    import torch.utils.pytree as pytree
```

Comment:
I"m confused, this doesn't exist yet. The PR should not be trying to import a module that doesn't exist

Reply:
It was cherrypicked from another PR stack. Reverted this change.